src/tslint-cli.ts

```diff
@@ -153,11 +153,24 @@ const processFile = (file: string) => {
         process.exit(1);
     }
 
+    const rulesDirectories: Array<string> = [];
+    if (configuration.rulesDirectory) {
+        if (typeof configuration.rulesDirectory === "string") {
+            rulesDirectories.push(configuration.rulesDirectory);
+        } else {
+            rulesDirectories.push(...configuration.rulesDirectory);
+        }
```

Comment:
this logic should go away since it's duplicated; use the new `public static` method on `Linter` which I propose below.